specifications/logging/default.yaml

```diff
@@ -0,0 +1,45 @@
+# this is a default logging configuration of cli
+
+# global filter; messages must have at least this severity to pass:
+minSeverity: Debug
+
+# here we set up outputs of logging in 'katip':
+setupScribes:
+  - scKind: StdoutSK
+    scFormat: ScText
+    scName: text
+  - scKind: StdoutSK
+    scFormat: ScJson
+    scName: json
+
+# if not indicated otherwise, then log output is directed to this:
+defaultScribes:
+  - - StdoutSK
+    - text
+
+# this is probably a bug of parsing, cannot enforce cgBindAddrPrometheus = Nothing
+#hasPrometheus: ["",0]
+#hasPrometheus:
+
+# these backends are initialized:
+setupBackends: [AggregationBK, KatipBK]
+
+# if not indicated otherwise, then messages are passed to these backends:
+defaultBackends:
+  - KatipBK
+
+# more options which can be passed as key-value pairs:
+options:
```

Comment:
This is needed in order to have set : 
```
cgMapSubtrace = fromList [("#messagecounters.monitoring",NoTrace),("#messagecounters.ekgview",NoTrace),("#messagecounters.aggregation",NoTrace),("#messagecounters.graylog",NoTrace),("#messagecounters.katip",NoTrace),("#messagecounters.switchboard",NoTrace)]
```
but it also sets : 
```
cgOptions = fromList [("mapSubtrace",fromList [("#messagecounters.monitoring",Object (fromList [("subtrace",String "NoTrace")])),("#messagecounters.ekgview",Object (fromList [("subtrace",String "NoTrace")])),("#messagecounters.aggregation",Object (fromList [("subtrace",String "NoTrace")])),("#messagecounters.graylog",Object (fromList [("subtrace",String "NoTrace")])),("#messagecounters.katip",Object (fromList [("subtrace",String "NoTrace")])),("#messagecounters.switchboard",Object (fromList [("subtrace",String "NoTrace")]))])],
```
which is maybe as it should be. BUT, it is expected in empty configuration that `cgOptions = fromList []` So it is inconsistent in such a sense, That empty is set like : https://github.com/input-output-hk/iohk-monitoring-framework/blob/master/iohk-monitoring/src/Cardano/BM/Configuration/Model.lhs#L543
and then on top of it is https://github.com/input-output-hk/iohk-monitoring-framework/blob/1ebe57df3a76a8ae4e3b5a31a9b85132f71534fc/iohk-monitoring/src/Cardano/BM/Configuration/Static.lhs#L26
So it behaves differently when yaml setting and in the code (this is modulo to my conclusions looking at the code)